thirdparty/moab/MOABConfig.new.cmake.in

```diff
@@ -50,14 +50,8 @@ set(MOAB_HAS_FBIGEOM "@ENABLE_IGEOM@")
 set(MOAB_HAS_IMESH "@ENABLE_IMESH@")
 set(MOAB_HAS_MPI "@MOAB_HAS_MPI@")
 
-if(MOAB_USE_HDF)
-  if(EXISTS "@HDF5_DIR@/share/cmake/hdf5/hdf5-config.cmake")
-    include(@HDF5_DIR@/share/cmake/hdf5/hdf5-config.cmake)
-  endif()
-endif()
```

Comment:
Well, comments upstream note that the `hdf5-config.cmake` file doesn't seem to even exist in their installs. So if it isn't needed, I suspect we can just rip it out.